src/olympia/files/utils.py

```diff
@@ -192,6 +199,10 @@ def parse(self, minimal=False):
             'version': self.find('version'),
             'is_webextension': False,
         }
+
+        if self.certinfo is not None:
```

Comment:
Add a comment explaining why it needs to be done even when `minimal` is `True` (we know now, but in 6 months...)